File: db_context/db_context/Customer.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace data_models
{
    public partial class Customer
    {
        public Customer()
        {
            Pets = new HashSet<Pet>();
            Posts = new HashSet<Post>();
        }

        public int CustomerId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }
        public double? HomeLocationLatitude { get; set; }
        public double? HomeLocationLongitude { get; set; }
        public double? WanderingRadius { get; set; }
        public int? Phone { get; set; }
        public int? ZipCode { get; set; }
        public DateTime AccountCreationDate { get; set; }

        public virtual ICollection<Pet> Pets { get; set; }
        public virtual ICollection<Post> Posts { get; set; }
    }
}

File: FindMyPetUI/src/app/customer/customer.ts
export class Customer
{

  constructor(
    public username: string = '',
    public password: string = '',
    public customerId: number = 0,
    public firstName: string = 'Test',
    public lastName: string = 'User',
    public email: string = '',
    public homeLongitude: number = 0,
    public homeLatitude: number = 0,
    public wanderingRadius: number = 0,
    public phoneNum: number = 0,
    public zipcode: number = 0
  ) {}

}